src/components/HeroSlider/HeroSlider.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import Slider from 'react-slick';
+
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import './HeroSlider.scss';
```

Comment:
We use Module CSS, so you must make some changes .
import styles from ''./HeroSlider.module.scss'';